proto/src/determined/api/v1/api.proto

```diff
@@ -2049,6 +2049,20 @@ service Determined {
       tags: "Workspaces"
     };
   }
+
+  // Gets the ids of all workspaces that are bound to the default namespace for
+  // atleast one cluster.
+  rpc GetWorkspacesWithDefaultNamespaceBindings(
```

Comment:
For some reason this API endpoint is erroring out with: 
```

> {
>   "error": {
>     "code": 3,
>     "reason": "InvalidArgument",
>     "error": "type mismatch, parameter: id, error: strconv.ParseInt: parsing \"default-namespace-bindings\": invalid syntax"
>   }
> }

```
Spent a bunch of time trying to figure out why but no luck. Would appreciate a second pair of eyes @corban-beaird @maxrussell

Reply:
I'm happy to pair! Just LMK on Slack whenever you're ready.

Reply:
fixed by changing path